taichi/backends/vulkan/vulkan_utils.h

```diff
@@ -15,30 +14,8 @@ namespace vulkan {
 class VulkanEnvSettings {
  public:
   static constexpr uint32_t kApiVersion() {
-    return VK_API_VERSION_1_2;
+    return VK_API_VERSION_1_0;
```

Comment:
Isn't this bit a part of fix with SPV version? #2603

Reply:
That was an accidental change. We have some clients preferring to use the 1.0 version. 

Just to confirm, does VK 1.0 work for you?

Reply:
It does not if the SPIRV code gen spits out SPV 1.5. The target SPV version for VK 1.0 should be SPV 1.2 iirc. (They need to match otherwise the shader will not load)

Reply:
https://github.com/taichi-dev/taichi/blob/70566a74f27766d73ea16e723182765d92118846/taichi/backends/vulkan/spirv_ir_builder.cpp#L16-L22
Now include a `spirv.hpp` header with `SPV_VERSION=0x10000` may work properly, but the following solution should be better:
https://github.com/apache/tvm/commit/019275075151a300a8d3aaa15db16be62da3327c#diff-a5248d0972171c59770042a098d0097a90452de88efacc8be7d49013ed34a322

Reply:
I can try if not in a hurry

Reply:
Related issue=https://github.com/taichi-dev/taichi/issues/2612